js/gulpfile.js

```diff
@@ -252,11 +239,16 @@ for (const name of tests) {
 
         let bundle = await rollup({
             input: input,
-            plugins: [IceResolver(), NodeMockupResolver()],
+            plugins: [IceResolver()],
             onwarn: (warning, next) => {
                 // Ignore the "this is undefined" warning, let rollup silently rewrite it.
                 // This avoids warnings from the TypeScript polyfills for async disposable resources.
                 if (warning.code === "THIS_IS_UNDEFINED") return;
+
+                // Ignore unresolved fs/path dynamic imports (Node-only usage)
+                if (warning.code === "UNRESOLVED_IMPORT" && ["fs", "path"].includes(warning.source)) {
```

Comment:
This is to avoid a warning in the properties test, which dynamically imports "fs" and "path"